client/src/views/users/UserGridDisplay.vue

```diff
@@ -6,6 +6,7 @@
     :user="currentUser as UserModel"
     :show-checkbox="false"
     :show-options="false"
+    :current-user="true"
```

Comment:
This prop does not exist on UserCard, it's called `isCurrentUser`. The `you` text is supposed to appear in grid-mode too. Update the test to also check that the current user has the `you` label.

Reply:
Done ✅